kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -183,6 +185,11 @@ public boolean importMotherRecord(Map<String, Object> record) {
             return false;
         }
 
+        //validate if it's an updated record compared to one from database
+        if (mother.getUpdatedDateNic() != null && (mctsUpdatedDateNic == null || mother.getUpdatedDateNic().isAfter(mctsUpdatedDateNic))) {
+            return true;
```

Comment:
Your #2 assumption is invalid. The rejected count has everything in a feed that didn't pass business rules to make it to the db. In this case, your update feed didn't make it into the db because it failed a new business logic (updated_date) that you have introduced. If it was up to me, I would merge this with the rejected count for the feed or introduce a new paradigm to track duplicate old updates. 

Returning true here breaks all existing logic and assumptions around counts.